mgrs.js

```diff
@@ -40,11 +40,13 @@ export default {
  * @param {object} ll Object literal with lat and lon properties on a
  *     WGS84 ellipsoid.
  * @param {int} accuracy Accuracy in digits (5 for 1 m, 4 for 10 m, 3 for
- *      100 m, 2 for 1000 m or 1 for 10000 m). Optional, default is 5.
+ *      100 m, 2 for 1000 m, 1 for 10000 m, or 0 for 100km). Optional, default is 5.
+ * @param {boolean} useCentroid If true, use the centroid of the resulting grid
+ * square to determine the latitude band.  If false, use ll.  False by default.
  * @return {string} the MGRS string for the given location and accuracy.
  */
-export function forward(ll, accuracy) {
-  accuracy = accuracy || 5; // default accuracy 1m
+export function forward(ll, accuracy, useCentroid) {
+  accuracy = typeof accuracy === 'number' ? accuracy : 5; // default accuracy 1m
```

Comment:
we probably need a check for a boolean being passed here, in which case that value will be swiched to the useCentroid and accuracy set to 5